### PR TITLE
WordPress 6.4.3 Release Note

### PR DESCRIPTION
**Pull request by pwtyler**
## Summary

**[Pantheon Release Notes](https://docs.pantheon.io/release-notes/)** - Adds release note for WordPress 6.4.3

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] Copy Editing
- [x] I'd like to include a link to the WordPress.org blog post in this post, but unsure of where/phrasing.

**Release**:
- [x] When ready

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/releasenotes/2024-01-31-wordpress-6-4-3.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+published_date: "2024-01-31"
+categories: [wordpress, security]
+---
+
+The latest version of WordPress, 6.4.3, became available on Pantheon as of January 30, 2024.
+
+<!--- // TODO insert link to https://wordpress.org/news/2024/01/wordpress-6-4-3-maintenance-and-security-release/ somewhere in here -->
+<h3>Highlights</h3>
+
+* **Security updates:** Addressed two security vulnerabilities, including a PHP file upload bypass (limited to administrators), and a object injection gadget that could be used to exploit an existing Remote Code Execution (RCE) vulnerability. For a detailed analysis of the two security patches, see [this article from Patchstack](https://patchstack.com/articles/wordpress-6-4-3-security-release/).
```
Comment (jazzsequence) [inline source/releasenotes/2024-01-31-wordpress-6-4-3.md:12]: > object injection gadget

I'm not sure what an "object injection gadget" is in this context. Can you use something else to describe the "gadget"?

**Review by jazzsequence — CHANGES_REQUESTED** [review]

File: source/releasenotes/2024-01-31-wordpress-6-4-3.md (hunk 1 above)
Comment (pwtyler) [reply source/releasenotes/2024-01-31-wordpress-6-4-3.md:12]: A mechanism to exploit another vulnerability is what it really means, but I agree it's still more jargon than I'd like I'll think on it.

Comment (pwtyler) [reply source/releasenotes/2024-01-31-wordpress-6-4-3.md:12]:
```suggestion
* **Security updates:** Addressed two security vulnerabilities, including a PHP file upload bypass (limited to administrators), and a object injection mechanism that could be used to exploit an existing Remote Code Execution (RCE) vulnerability. For a detailed analysis of the two security patches, see [this article from Patchstack](https://patchstack.com/articles/wordpress-6-4-3-security-release/).
```

File: source/releasenotes/2024-01-31-wordpress-6-4-3.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: WordPress 6.4.3 Security Updates
+published_date: "2024-01-31"
+categories: [wordpress, security]
+---
+
+The latest version of WordPress, 6.4.3, became available on Pantheon as of January 30, 2024.
```
Comment (jazzsequence) [inline source/releasenotes/2024-01-31-wordpress-6-4-3.md:7]: link the WP release notes here
```suggestion
The latest version of WordPress, [6.4.3](https://wordpress.org/news/2024/01/wordpress-6-4-3-maintenance-and-security-release/), became available on Pantheon as of January 30, 2024.
```

File: source/releasenotes/2024-01-31-wordpress-6-4-3.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+<!--- // TODO insert link to https://wordpress.org/news/2024/01/wordpress-6-4-3-maintenance-and-security-release/ somewhere in here -->
```
Comment (jazzsequence) [inline source/releasenotes/2024-01-31-wordpress-6-4-3.md:9]: obvs drop TODO
```suggestion
```

**Review by jazzsequence — APPROVED** [review]

**Pull request merged**